examples/spark2/src/main/scala/org/apache/carbondata/examples/ExampleUtils.scala

```diff
@@ -56,6 +56,7 @@ object ExampleUtils {
       .master(masterUrl)
       .appName(appName)
       .config("spark.sql.warehouse.dir", warehouse)
+      .config(CarbonCommonConstants.CARBON_COMMON_LISTENER_REGISTER_CLASSNAME, "")
```

Comment:
instead of adding it to all the example classes add a try catch in
org.apache.spark.sql.CarbonEnv#registerCommonListener to get the clsName
if exception thrown, make it as null or empty

Reply:
@dhatchayani please review